pkg/consuldp/config.go

```diff
@@ -297,6 +297,11 @@ type EnvoyConfig struct {
 	GracefulPort int
 	// DumpEnvoyConfigOnExitEnabled configures whether to call Envoy's /config_dump endpoint during consul-dataplane controlled shutdown.
 	DumpEnvoyConfigOnExitEnabled bool
+	//##############
```

Comment:
**issue:** Same as above. Please move this up a few lines so that it is with the other graceful flags. Also, please remove //########